gatorgrade/generate/generate.py

```diff
@@ -14,8 +14,40 @@ def create_targeted_paths_list():
     # Add those targeted file paths into a list and returns it.
 
 
-def write_yaml_of_paths_list():  # expected input: A path list
+def write_yaml_of_paths_list(path_names):  # expected input: A path list
     """Write YAML file to create gatorgrade file and set default messages."""
-    # Create a new YAML file with PyYaml in the specific path.
+
+    setup_dict = {
+        "setup": "# add setup commands here\n",
+    }
     # Write the default set up messages in YAML file.
-    # List the file paths in specific format.
+
+    path_names = create_targeted_paths_list(["test"])
```

Comment:
This fine is confusing to me. The variable `path_names` was used as an input argument, but here you re-define it again. Also, I don't think it's a good idea to call another function in this function. It should be the job of function `generate_config`

Reply:
You're right, thanks!